deployment/articles/03-installing-a-search-engine/01-intro.markdown

```diff
@@ -21,39 +21,76 @@ Learn to configure a remote Elasticsearch server or cluster
 [Solr](http://lucene.apache.org/solr) 
 is another capable and popular search engine supported in @product@. 
 
-<!--Learn to configure a remote Solr server or cluster 
-here(/docs/7-2/deploy/-/knowledge_base/d/installing-solr). 
--->
+Learn to configure a remote Solr server or cluster
+[here](/docs/7-2/deploy/-/knowledge_base/d/installing-solr). But first, make
+sure you understand the disparity in functionality between the supported search
+engines.
 
 ## Choosing a Search Engine
 
-Elasticsearch and Solr are both supported, but there are some differences in how
-they work with @product@. In certain cases, you must choose Elasticsearch. 
-
-If you answer _yes_ to either of these questions, you must choose Elasticsearch:
-
-1.  You're using @commerce@.
-
-2.  Your custom search code requires the use of the `TermsSetFilter` API or the
-    Geolocation APIs that are implemented in the Liferay Connector to
-    Elasticsearch.
-
-@commerce@ requires the `TermsSetFilter` implementation, only available in the
-Elasticsearch connector.
-
-Both of these Elasticsearch-only developer features are not currently
-implemented in the Liferay Connector to Solr, but may be added in the future.
-If you must use either of those features in your search solution's code, use
-Elasticsearch. If you're using @commerce@, use Elasticsearch.
-Otherwise, feel free to use Elasticsearch or Solr to index your portal content.
+Elasticsearch and Solr are both supported, but there are limitations to
+Liferay's Solr integration. To make use of some features, you must choose
+Elasticsearch. 
+
+### End User Feature Limitations of Liferay's Solr Integration
+
+- [Liferay Commerce](https://help.liferay.com/hc/en-us/articles/360017869952)
+- [Workflow Metrics](/docs/7-2/user/-/knowledge_base/u/workflow-metrics-the-service-level-agreement) 
+- [Custom Filter search widget](/docs/7-2/user/-/knowledge_base/u/filtering-search-results-with-the-custom-filter-widget)
```

Comment:
@lipusz do we need to add the federated search/Low Level Search Options widget here?